### PR TITLE
mm_initialize: malloc() return aligend pointer. 

### PR DESCRIPTION
**Pull request by CV-Bowen**
## Summary
malloc() should return aligned (with MM_MIN_CHUNK) pointer, but pr https://github.com/apache/incubator-nuttx/pull/5906 destroy that, this pr find a better method to solve these problems.

## Testing
mm test pass with nucleo-144:f46zg and sim (CONFIG_DEBUG_MM=y/n, CONFIG_MM_SMALL=y/n)

@xiaoxiang781216

### CODE REVIEW

File: mm/mm_heap/mm_initialize.c [hunk 1]
```diff
@@ -97,7 +97,8 @@ void mm_addregion(FAR struct mm_heap_s *heap, FAR void *heapstart,
    * with the MM_MIN_CHUNK size.
```
Comment (xiaoxiang781216) [inline mm/mm_heap/mm_initialize.c:97]: let's copy the comment from https://github.com/apache/incubator-nuttx/pull/5991

Comment (CV-Bowen) [reply mm/mm_heap/mm_initialize.c]: Done

**Review by pkarashchenko — APPROVED** [review]

**Review by yamt — CHANGES_REQUESTED** [review]
please add my signed-off-by for the comment you copied.
```
Signed-off-by: YAMAMOTO Takashi <yamamoto@midokura.com>
```
otherwise lgtm.

**Comment by CV-Bowen** [comment]
@yamt Done

**Comment by yamt** [comment]
> @yamt Done

i meant in the commit message as usual

**Comment by CV-Bowen** [comment]
> > @yamt Done
> 
> i meant in the commit message as usual

i misunderstood, done.

**Review by yamt — APPROVED** [review]
lgtm

**Review by xiaoxiang781216 — APPROVED** [review]

**Pull request merged**